docker/images/clueso-docker-livy/entrypoint.sh

```diff
@@ -16,4 +17,16 @@ else
   export LIBPROCESS_IP=$HOST
 fi
 
+# wait for Spark master before starting livy
+echo "Waiting for Spark Master to launch on 7077..."
+
+if [ ! nc -z spark-master 7077 &> /dev/null ]; then
+  echo 'Cannot contact spark-master on 7077' && exit 1
+fi
+
+echo "Spark Master Ready"
+sleep 3
```

Comment:
Any reason for this one?

Reply:
Remnant from debugging, I'll get rid of it as well.